src/sound_openal.cpp

```diff
@@ -457,18 +536,10 @@ class OpenALSoundManager: public ISoundManager
 	bool loadSoundData(const std::string &name,
 			const std::string &filedata)
 	{
-		// The vorbis API sucks; just write it to a file and use vorbisfile
-		// TODO: Actually load it directly from memory
-		std::string basepath = porting::path_user + DIR_DELIM + "cache" +
-				DIR_DELIM + "tmp";
-		std::string path = basepath + DIR_DELIM + "tmp.ogg";
```

Comment:
Perhaps add compat code to delete this directory?  (this is all it's used for, probably should have been `/tmp` anyway)